crates/holochain_zome_types/src/fixt.rs

```diff
@@ -359,7 +359,7 @@ fixturator!(
             CapAccessVariant::Unrestricted => CapAccess::from(()),
             CapAccessVariant::Transferable => CapAccess::from(CapSecretFixturator::new_indexed(Empty, get_fixt_index!()).next().unwrap()),
             CapAccessVariant::Assigned => CapAccess::from((
-                CapSecretFixturator::new_indexed(Empty, get_fixt_index!()).next().unwrap(),
+                Some(CapSecretFixturator::new_indexed(Empty, get_fixt_index!()).next().unwrap()),
```

Comment:
Would it be more semantically correct to use `None` for an empty fixture? (I have no clue; just applying beginner's mind here)

Reply:
Yeah, that would probably be better